addons/mail/static/src/core/common/thread_model.js

```diff
@@ -279,6 +279,8 @@ export class Thread extends Record {
     custom_notifications = false;
     /** @type {String} */
     mute_until_dt;
+    /** @type {Boolean} */
+    membersFetched = false;
```

Comment:
suggesting to name `hasFetchedMembers` to clearly indicate it is a boolean (current name could imply it contains the fetch members)